src/content/docs/reference/configuration.md

```diff
@@ -201,29 +201,71 @@ export default {
 
 ### Solidity tests configuration
 
-By default, Hardhat includes support for tests written in Solidity to test your contracts. You can use the `test.solidity` entry to configure how they behave. For example, the following config enables the `ffi` cheatcode:
+By default, Hardhat includes support for tests written in Solidity to test your contracts. You can use the `test.solidity` entry to configure how they behave. For example, the following config enables `isolate` mode:
 
 ```js
 const config = {
   test: {
     solidity: {
-      ffi: true,
+      isolate: true,
```

Comment:
I changed the example to `isolate`, because `ffi` is dangerous and we don't want to encourage users to use it.